tests/utils.py

```diff
@@ -21,6 +21,7 @@ def setup_task(
         overwrite_from_index_token=None,
         state_seq=("in_progress", "finished"),
         op_type="add",
+        deprecation_list=None,
```

Comment:
I am just wondering why does it accepts the parameter when it is not used.
I know that there are others unused parameters as well. But do we really want to be empty every time?

should not line 49  look like this ` "deprecation_list": [] if not deprecation_list else deprecation_list`?

Reply:
solved